idea.js

```diff
@@ -13,15 +13,18 @@ class Idea {
 
 	deleteFromStorage(index, array) {
 		var newArray = array.splice(index, 1);
-		
-		console.log(newArray);
-		console.log(id)
 		this.saveToStorage(ideasArray);
 	}
 
-	updateIdea(title, body) {
+	updateIdea(title, body, star) {
 		this.title = title;
 		this.body = body;
+		this.star = star;
+		console.log(this.star);
+	}
```

Comment:
We don't need to update the star property in this method. Can you remove star from the parameters and also remove line 22 & 23?